src/components/cms/MyDevices/MapContainer.js

```diff
@@ -62,44 +66,48 @@ class MapContainer extends Component {
     }
   };
 
-  getMapCenter = (devicesData, invalidLocationDevices) => {
-    const latitudeSum = devicesData.reduce(
+  getMapCenter = (data, invalidLocationDevices) => {
+    const latitudeSum = data.reduce(
       (a, b) => ({ latitude: a.latitude + b.latitude }),
       {
         latitude: 0.0,
       },
     ).latitude;
-    const longitudeSum = devicesData.reduce(
+    const longitudeSum = data.reduce(
       (a, b) => ({ longitude: a.longitude + b.longitude }),
       { longitude: 0.0 },
     ).longitude;
 
     return {
-      lat: latitudeSum / (devicesData.length - invalidLocationDevices),
-      lng: longitudeSum / (devicesData.length - invalidLocationDevices),
+      lat: latitudeSum / (data.length - invalidLocationDevices),
+      lng: longitudeSum / (data.length - invalidLocationDevices),
     };
   };
 
   render() {
     const { activeMarker, showingInfoWindow, selectedPlace } = this.state;
     const {
       google,
-      devicesData = [],
-      invalidLocationDevices,
+      data = [],
+      invalidLocationDevices = 0,
```

Comment:
Rename this variable also to make it generic.

Reply:
I have renamed it to invalidLocations .